packages/core/src/Stores/ui-store.js

```diff
@@ -227,6 +227,24 @@ export default class UIStore extends BaseStore {
         return !!this.account_switcher_disabled_message;
     }
 
+    @action.bound
+    filterNotificationMessages() {
+        this.notifications = this.notification_messages.filter(notification => {
+            if (notification.platform === undefined) {
+                return true;
+            } else if (notification.platform && notification.platform.includes(getPathname())) {
```

Comment:
We've already determined `notification.platform` is not undefined, so we can leave out the first condition.

```suggestion
            } else if (notification.platform.includes(getPathname())) {
```

Reply:
But `notification.platform` is undefined for all other existing notifications that don't have the platform property. So when the notification centre has to be filtered based on platform but there are notifications where platform isn't defined, they're still returned.